### PR TITLE
Add section on enforcing the code of conduct

### PR DESCRIPTION
**Pull request by caniszczyk**
There should be a way to report issues in regards to the code of conduct. It's great that we currently have a code of conduct that we expect our community to follow but we need a way to report issues.

### CODE REVIEW

**Comment by caniszczyk** [comment]
Let me know if you have any questions @opencontainers/tob

File: code-of-conduct.md [hunk 1]
```diff
@@ -30,6 +30,12 @@ The OpenContainers team reserves the right to deny participation any individual
 Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct.
 By adopting this Code of Conduct, project maintainers commit themselves to fairly and consistently applying these principles to every aspect of managing this project.
 
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the code of conduct team at codeofconduct@opencontainers.org. All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. The code of conduct team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (wking) [inline code-of-conduct.md]: codeofconduct@opencontainers.org → conduct@opencontainers.org.  Do other orgs really use “codeofconduct@”?

**Comment by wking** [comment]
Excepting the long email user nit, b9db2060 looks good to me.

File: code-of-conduct.md (hunk 1 above)
Comment (srpanoplos) [inline code-of-conduct.md]: This seems like a partial code of conduct -- have seen other code of conducts which include this language as well.. "in the interest of fostering an open and welcoming community, we pledge to respect all people who contribute through reporting issues, making comments, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.."

Comment (wking) [inline code-of-conduct.md]: On Tue, Jun 07, 2016 at 10:16:38PM -0700, srpanoplos wrote:
> This seems like a partial code of conduct -- have seen other code of
> conducts which include this language as well.. "in the interest of
> fostering an open and welcoming community, we pledge to respect all
> people who contribute through reporting issues, making comments,
> posting feature requests, updating documentation, submitting pull
> requests or patches, and other activities.."

Both the Fedora and Contributor Covenant examples we cite as
inspirational also have this sort of positive goal-setting as well, so
I'm +1 to including something like it.  However, it seems orthogonal
to the violation-reporting and enforcement language that this PR is
about; maybe file a separate PR?

**Comment by vbatts** [comment]
LGTM

**Comment by philips** [comment]
LGTM

File: code-of-conduct.md (hunk 1 above)
Comment (caniszczyk) [inline code-of-conduct.md]: I agree with @wking , I'd recommend filing another issue or PR around this, thanks!
